app/schemas.py

```diff
@@ -0,0 +1,31 @@
+from pydantic import BaseModel
+from datetime import datetime
+from typing import Optional
+
+class CityBase(BaseModel):
+    name: str
+    additional_info: Optional[str] = None
+
+class CityCreate(CityBase):
+    pass
+
+class City(CityBase):
+    id: int
+
+    class Config:
+        orm_mode = True
+
+class TemperatureBase(BaseModel):
+    city_id: int
+    date_time: datetime

```

Comment:
The `date_time` field in `TemperatureBase` does not have a default value. Consider setting a default value, such as `datetime.utcnow()`, if this field is not always provided by the client.